src/controller/controller.cpp

```diff
@@ -217,6 +217,20 @@ NWNXController::NWNXController(SimpleIniConfig* config)
 			return;
 		}
 	}
+
+	// TODO: This has a far reaching effect. Need more time to test out repercussions of using long paths
+	/*#ifdef _WIN32
+	// Win32 applications have problems with long paths (exceeding 260 characters)
+	const char* LONG_PATH_PREFIX = R"(\\?\)";
+	const int LONG_PATH_PADDING = 64;
+
+	if (nwninstalldir.length() >= MAX_PATH - LONG_PATH_PADDING &&  // If the length exceeds the MAX_PATH without its padding
+		nwninstalldir.rfind(LONG_PATH_PREFIX, 0) == std::string::npos)  // If the prefix isn't already found on the install directory
+	{
+		nwninstalldir = LONG_PATH_PREFIX + nwninstalldir;
+	}
+	#endif*/
+
```

Comment:
I think this can be safely dropped: supporting long executable paths isn't very useful and is likely to bring compatibility issues with nwn2server